tools/dptools/src/dptools/scripts/repeatgen.py

```diff
@@ -125,3 +136,35 @@ def _repeatgeo(geo, latvecs, repeats):
     newgeo = Geometry(geo.specienames, allinds, allcoords, latvecs=newlatvecs,
                       origin=geo.origin)
     return newgeo
+
+def _repeatgeo2(geo, latvecs, repeats):
```

Comment:
maybe we can merge this into the original routine...

Reply:
_repeatgeo and _repeatgeo2 can be merged. 
Is it fine to pass option.phonons to the subroutine ?

Reply:
there is one small issue: in repeatgen the loops are in order [0], [1], [2], whereas in geo2 they are in reverse, [2],[1],[0]. So I'll have to duplicate the core of the subroutine. 
Still like this ?